assets/js/WorkflowEditor/wfmodule/StatusLine.js

```diff
@@ -46,7 +46,7 @@ const StatusLine = React.memo(function StatusLine ({ status, errors, applyQuickF
 StatusLine.propTypes = {
   status: PropTypes.oneOf(['ok', 'busy', 'error', 'unreachable']).isRequired,
   isReadOnly: PropTypes.bool.isRequired, // if true, cannot apply quick fixes
-  errors: PropTypes.arrayOf(PropTypes.shape({ message: PropTypes.string.isRequired, quickFixes: PropTypes.arrayOf(PropTypes.shape(QuickFixPropTypes)) })), // may be empty
+  errors: PropTypes.arrayOf(PropTypes.shape({ message: PropTypes.string.isRequired, quickFixes: PropTypes.arrayOf(PropTypes.shape(QuickFixPropTypes)).isRequired }).isRequired).isRequired, // may be empty
```

Comment:
Please split this onto multiple lines. It's _still_ missing an `.isRequired`. (`PropTypes.shape(QuickFixPropTypes).isRequired`)

```
errors: PropTypes.arrayOf(
    PropTypes.shape({
        message: PropTypes.string.isRequired,
        quickFixes: PropTypes.arrayOf(PropTypes.shape(QuickFixPropTypes).isRequired).isRequired
    }).isRequired
).isRequired
```